host_vars/pulsar-paw/pulsar-paw.genome.edu.au.yml

```diff
@@ -13,7 +13,7 @@ ssl_email: "help@genome.edu.au"
 #host specific pulsar settings
 
 rabbitmq_password_galaxy_au: "{{ vault_rabbitmq_password_galaxy_paw_pawsey }}"
```

Comment:
```suggestion
rabbitmq_password_galaxy_au: "{{ vault_rabbitmq_password_galaxy_paw_prod }}"
```

Reply:
there are two sets of passwords in the vault that are the same as one another.  I've been using the prod values instead of spinning up new ones for aarnet.